src/components/NewsletterFeature.vue

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+// libraries
+import { defineComponent } from 'vue';
+
+// mocks
+import { newsletterItems } from 'src/mocks/homepage';
+
+// components
+import NewsletterItem from 'components/NewsletterItem.vue';
+
+export default defineComponent({
+  name: 'NewsletterFeature',
+  components: {
+    NewsletterItem,
+  },
+  setup() {
+    return {
+      newsletterItems,
+    };
+  },
+});
+</script>
+
+<template>
+  <div
+    data-cy="newsletter-feature"
+    class="row q-col-gutter-lg items-center justify-between"
+  >
+    <!-- Section image -->
+    <div class="gt-sm col-md-3" data-cy="newsletter-col-image">
+      <div class="q-px-xl">
+        <!-- Image -->
+        <q-img
+          src="~assets/image/newsletter-envelope.png"
```

Comment:
Newsletter envelope image should be vector SVG image file format. The icon seems too big to me.

Reply:
Swapped assets and updated image size - column width and <img> max-width